blockrecord/validator.go

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: ISC
+// Copyright (c) 2014-2019 Bitmark Inc.
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package blockrecord
+
+import (
+	"github.com/bitmark-inc/bitmarkd/blockdigest"
+	"github.com/bitmark-inc/bitmarkd/difficulty"
+	"github.com/bitmark-inc/bitmarkd/fault"
+)
+
+const (
+	blockTimeSpacingInitial = 240 * 60
```

Comment:
I think you should leave comment for determining the unit here.

Reply:
ok, i will rename the variable